modules/nf-core/wipertools/fastqwiper/main.nf

```diff
@@ -0,0 +1,51 @@
+process WIPERTOOLS_FASTQWIPER {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/wipertools:1.1.3--pyhdfd78af_0':
+        'biocontainers/wipertools:1.1.3--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fastq_in)
+
+    output:
+    tuple val(meta), path("${prefix}.fastq.gz"), emit: fastq_out
+    path("*.report")                           , emit: report
+    path "versions.yml"                        , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args    = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}_wiped"
+    if ("${fastq_in}" == "${prefix}.fastq.gz") error "Input and output names are the same, set prefix in module configuration to disambiguate!"
```

Comment:
I'm not sure about this one, how many reads does this take in? Does it take paired reads? The -o output suggests that it only outputs one file, but please check.

Reply:
No, it process one fastq file in input and outputs one fastq file in output. It should be fine.
I may shorten the error message to `Input and output names are the same, use \"task.ext.prefix\" to disambiguate!`

Reply:
Sure, feel free to shorten it (and the ones in the other submodules).